### PR TITLE
Emphasize Openverse

### LINKED ISSUE #108: [Feature] Emphasize Openverse
## Problem
Openverse is a project closely tied to Creative Commons. It is good that we are currently visually emphasizing it with a banner. However, once #37 and #104 are completed, the banner will no longer exist.

## Description
Once the issues noted above are completed, we should find a new way to visually emphasize Openverse  (maybe a thicker button border, bold text, etc.).

Additionally (or alternatively), we might also remove the random engine selection and instead always default to Openverse.

### Current Banner
Screen Shot 2022-10-19 at 08 00 43

## Additional context
- #104
- #37 
- 2021-12-13: [Dear Users of CC Search, Welcome to Openverse - Creative Commons](https://creativecommons.org/2021/12/13/dear-users-of-cc-search-welcome-to-openverse/)
- 2021-05-10: [Welcome to Openverse – Openverse — WordPress.org](https://make.wordpress.org/openverse/2021/05/11/hello-world/)
- 2021-05-03: [CC Search to Join WordPress - Creative Commons](https://creativecommons.org/2021/05/03/cc-search-to-join-wordpress/)
- 2020-12-07: [Upcoming Changes to the CC Open Source Community — Creative Commons Open Source](https://opensource.creativecommons.org/blog/entries/2020-12-07-upcoming-changes-to-community/)

### PR DESCRIPTION
**Pull request by ImaJin14**
## Fixes

Fixes #108 by @TimidRobot  and #97 by @JennySimen 
## Description

Defaults search engine to openverse, increase openverse to bold text. This resolves the issue on #97 as there is always a selected card which is the default ```openverse```
## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by ImaJin14** [comment]
@possumbilities please do a review:)

**Comment by possumbilities** [comment]
@ImaJin14 PR seems to be out of date with main

**Comment by ImaJin14** [comment]
@possumbilities branch been brought up to date

**Comment by JennySimen** [comment]
@ImaJin14 can you please tag the assignee working on issue #97 and also add a short note on how your PR fixes the ticket as well so that the person working on it can better understand your fix and they can move on to another ticket when this is merged?

**Comment by ImaJin14** [comment]
@JennySimen issue #97 was to address the fact that there where instances where an engine wasn't selected at random but with default set to openverse there will always be a selected engine......

**Comment by ImaJin14** [comment]
@possumbilities please do a review!...

**Review by possumbilities — COMMENTED** [review]
I added some comments, but after reviewing the code I'm not sure this code does what it is supposed to do. Most of your changes (even if incorrect) are in comments which would have no affect.

I'd suggest you review and test your code again to verify that it does in fact work as outlined.

File: search.js [hunk 1]
```diff
@@ -270,7 +270,7 @@ function setEngine(e) {
 	$(".engine").removeClass("selected");
 	$("button[value=" + engine + "]").parents(".engine").addClass("selected");
 
-	//if (e == "_random") engine = "_random";
+	//if (e == "epenverse") engine = "openverse";
```
[the remark this answers is not in the document]
Comment (ImaJin14) [reply search.js:273]: I'll do correct the spelling thanks...

**Comment by ImaJin14** [comment]
> I added some comments, but after reviewing the code I'm not sure this code does what it is supposed to do. Most of your changes (even if incorrect) are in comments which would have no affect.
> 
> I'd suggest you review and test your code again to verify that it does in fact work as outlined.

The PR does do what it is outlined for... I've tested it several times.... Also I'll love to push one last commit with the right spelling for openverse....

File: search.js [hunk 2]
```diff
@@ -254,7 +254,7 @@ function setEngine(e) {
 
 	if (typeof e == "string") {
 		if (e == "_random") {
-			engine = engines[Math.floor(Math.random() * engines.length)];
+			engine = engines[Math.floor(Math.openverse() * engines.length)];
```
[the remark this answers is not in the document]
Comment (possumbilities) [reply search.js:257]: To elaborate further, this function is grabbing all the engines, and using a random function picking one each time the page is reloaded. If the engines are always going to be set to openverse then the entire function is no longer necessary and instead the default engine can simply be selected outright without any need to generate a random outcome. 

where you replaced `random` with `openverse` is not a valid function to append to `Math` object. See here for further details: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random

[the remark this answers is not in the document]
Comment (ImaJin14) [reply search.js]: > To elaborate further, this function is grabbing all the engines, and using a random function picking one each time the page is reloaded. If the engines are always going to be set to openverse then the entire function is no longer necessary and instead the default engine can simply be selected outright without any need to generate a random outcome.
> 
> where you replaced `random` with `openverse` is not a valid function to append to `Math` object. See here for further details: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random

Thanks for the heads up... I've removed the block of code in question

**Comment by ImaJin14** [comment]
@possumbilities I believe I've resolved all code bugs... PR is now ready for review....

**Comment by possumbilities** [comment]
@ImaJin14 Tested via Docker, if I type in a search term and hit the `Enter` key it uses ccMixter instead. Seems there's missing functionality.

**Pull request closed without merging**